doc/guide/transactions-and-threading.rst

```diff
@@ -0,0 +1,394 @@
+============================
+Transactions and concurrency
+============================
+
+.. contents::
+
+`Transactions <https://en.wikipedia.org/wiki/Database_transaction>`_
+are a core feature of ZODB.  Much has been written about transactions,
```

Comment:
A footnote or callout that the transaction package is separate and can be (and is) used by other things? It's a nice package.